eox_tenant/settings/aws.py

```diff
@@ -43,3 +49,8 @@ def plugin_settings(settings):  # pylint: disable=function-redefined
             'eox_tenant.middleware.AvailableScreenMiddleware',
             'eox_tenant.middleware.MicrositeCrossBrandingFilterMiddleware',
         ]
+
+        if EDX_AUTH_BACKEND in settings.AUTHENTICATION_BACKENDS:
```

Comment:
I wonder if we should replace the EDX_AUTH_BACKEND in the same exact position (list index) on AUTHENTICATION_BACKENDS, and if the EDX_AUTH_BACKEND is not in the AUTHENTICATION_BACKENDS list, just append it

Reply:
I am ok with replace in the same exact position, but I am against append that backend if the EDX_AUTH_BACKEND is not in the AUTHENTICATION_BACKENDS .  

If an openedx instance use CAS, it would be undesired to append that backend.

Reply:
@morenol you're right, if the expected default backend is not present, don't do anything